internal/tui/README.md

```
@@ -387,19 +387,32 @@ tui.NewScrollbar(tui.ScrollbarOptions{Total: 30, Visible: 5, Offset: 10, Height:
+    tui.TableCell{Value: true, TerminalText: tui.GreenText.Render("Yes")},
+)
+
+format, err := tui.ParseTableFormat(formatFlag)
+err = table.Write(cmd.OutOrStdout(), format)
 ```
 
+`TableCell` keeps the typed JSON value separate from styled or human-readable
+terminal text. `RenderTable` and `PrintTable` remain available for terminal-only
+call sites using `[][]string`.
+
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)